R/Analyis_genome_abundances.Rmd

```diff
@@ -0,0 +1,215 @@
+---
+title: "Metagenome-atlas output summary"
+output: html_document
```

Comment:
hmm I think this can be accomplished with github actions, although I've never set that up. I know @luizirber recently set up github actions to build a pdf of an Rmd every time he pushed changes to his thesis...I'm linking that repo here.
https://github.com/luizirber/phd/actions

Reply:
I refuse to participate in a PR that is replacing ipynb with RMarkdown.

(just kidding =])

I can help set up GH actions to rebuild HTML (and also prebuild Binder images), are you publishing the HTML to github pages?